### PR TITLE
pre-commit improvements

### PR DESCRIPTION
**Pull request by t-young31**

### CODE REVIEW

File: .github/workflows/clean_validation_envs.yml [hunk 1]
```diff
@@ -15,9 +15,8 @@
 name: TODO
 
 on:
-  schedule:
-    # Every 2 hours
-    - cron: "0 */2 * * *"
+  # schedule:
+  #  - cron: "0 */2 * * *"  # Every 2 hours
```
Comment (t-young31) [inline .github/workflows/clean_validation_envs.yml:19]: will be enabled in a future PR!

File: README.md [hunk 2]
```diff
@@ -36,28 +36,28 @@ FlowEHR is a safe, secure &amp; cloud-native development &amp; deployment platfo
 
 
 2. <details>
-    <summary>Copy and edit the sample environment file</summary>
-   
-    Local deployment i.e. non CI/CD requires a `.env` file. Copy and edit as appropriate.
+    <summary>Copy and edit the sample configuration file</summary>
```
Comment (jjgriff93) [inline README.md:39]: Thanks for updating :)

File: infrastructure/main.tf [hunk 3]
```diff
@@ -1,4 +1,6 @@
 terraform {
+  required_version = "0.13"
```
Comment (jjgriff93) [inline infrastructure/main.tf:2]:
```suggestion
  required_version = "1.3.7"
```

File: infrastructure/core/main.tf [hunk 4]
```diff
@@ -1,3 +1,15 @@
+terraform {
+  required_version = "0.13"
```
Comment (jjgriff93) [inline infrastructure/core/main.tf:2]:
```suggestion
  required_version = "1.3.7"
```

**Review by jjgriff93 — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (t-young31) [reply README.md:39]: nw 😄

File: infrastructure/main.tf (hunk 3 above)
Comment (t-young31) [reply infrastructure/main.tf:2]: 🤦🏼

Comment (t-young31) [reply infrastructure/main.tf:2]:
```bash
vscode ➜ /workspaces/FlowEHR (t-young31/scaffolding) $ terraform --version
Terraform v1.3.6
on linux_arm64
```
can we pin to 1.3 ?

Comment (jjgriff93) [reply infrastructure/main.tf:2]: Just updated the dc to 1.3.7, but yeah we can pin more generically

**Pull request merged**